app/controllers/concerns/hyrax/admin/users_controller_behavior.rb

```diff
@@ -0,0 +1,25 @@
+module Hyrax
+  module Admin
+    module UsersControllerBehavior
+      extend ActiveSupport::Concern
+
+      included do
+        include Blacklight::SearchContext
+      end
+
+      # Display admin menu list of users
+      def index
+        add_breadcrumb t(:'hyrax.controls.home'), root_path
+        add_breadcrumb t(:'hyrax.toolbar.admin.menu'), hyrax.admin_path
+        add_breadcrumb t(:'hyrax.admin.users.index.title'), hyrax.admin_users_path
+        @presenter = Hyrax::Admin::UsersPresenter.new
+      end
+
+      protected
+
+        def deny_access(_exception)
```

Comment:
This was a carryover from the controller that I began with from Hyku. I have removed it.